### DISCUSSION TITLE
How to schedule in Unraid?

### DISCUSSION THREAD

**Discussion by sero2kk** [vogler/free-games-claimer#246] [Q&A] [answered]
Docker Compose command:       `bash -c "node epic-games; node prime-gaming; node gog; echo sleeping; sleep 1d"`
additionally add                           `restart: unless-stopped`       to it.

How do I set up a schedule with Unraid?

**Comment by Lifeng77X** [comment]
> Commande Docker Compose : ajoutez-y en plus.`bash -c "node epic-games; node prime-gaming; node gog; echo sleeping; sleep 1d"``restart: unless-stopped`
> 
> Comment puis-je configurer un calendrier avec Unraid ?

Hello, have you found the answer? I've been trying for several weeks.

**Reply by sero2kk** [reply, marked as answer]
Go to “Apps” and install “User Scripts”. After installing “User Scripts”, you will see a new option under Settings > User Utilities called “User Scripts”.

You can set up the rest there.

![Screenshot_7 png 2cec093cb8f326317315c356e4fb18d6](https://github.com/user-attachments/assets/13a62d23-1853-4e1c-bf12-60412c94200d)



![Screenshot_56](https://github.com/user-attachments/assets/fb0bea8e-f628-4f36-9c57-edde57931791)



![Screenshot_56](https://github.com/user-attachments/assets/557f6c85-62aa-4309-b8dc-340fc75a5576)



![image png 900b0b1a973b2d80770969a5ed327827](https://github.com/user-attachments/assets/367904a4-eacf-4117-a714-9e2ab729f0f2)
